src/intents/total_steps_intent.js

```diff
@@ -26,7 +25,7 @@ export async function handle_get_num_remaining_steps(req,res,sessionData, contex
 
     res.status(201);
     response.fulfillmentText = response_text;
-    response.outputContexts = set_session_data(contexts, sessionData, projectID, sessionID);
+    response.outputContexts = data.queryResult.outputContexts;
```

Comment:
This will overwrite how we're handling session data for dialogflow
```suggestion
    response.outputContexts = set_session_data(contexts, sessionData, projectID, sessionID);
```